renderdoc/driver/vulkan/vk_posix.cpp

```diff
@@ -55,6 +55,17 @@ bool WrappedVulkan::AddRequiredExtensions(bool instance, vector<string> &extensi
       extensionList.push_back(VK_KHR_SURFACE_EXTENSION_NAME);
 
     bool oneSurfaceTypeSupported = false;
+ 
+#if defined(VK_USE_PLATFORM_ANDROID_KHR)
```

Comment:
Sorry, this was a fix I've been dragging along for a while.  I didn't see the recent changes you made.  I'll look again, since without my changes, this file didn't compile.